### DISCUSSION TITLE
Remove `lxqt-qt5ct` from recommended third party apps — and more...

### DISCUSSION THREAD

**Discussion by tsujan** [lxqt/lxqt#2223] [General]
It's listed in https://github.com/lxqt/lxqt/wiki/Third-party-apps but, apart from being very old, it clashes with `lxqt-config-appearance` and could confuse the user.

Actually, we should somehow tell our users that third-party Qt plugins — including `qt5ct` and soon, `qt6ct` — are in conflict with LXQt's Qt plugins and, if they're activated under LXQt, inconsistencies will arise. Of course, it doesn't mean that those third-party plugins are "bad", but they're written for DEs that lack good Qt plugins (especially, GTK-based DEs).

My other suggestion is giving more importance to @palinek's `nm-tray`. Most people don't use connman.

**Comment by tsujan** [comment]
Also, we may need to tell more about GVFS. It's crucial to a good file management and, although `libfm-qt` doesn't depend on it, it's needed for having "elementary" functionalities like Trash support as well as more advanced properties like emblems.

IMHO, https://github.com/lxqt/lxqt/wiki/Third-party-apps is a good place for explaining these things.

EDIT: I emphasized the importance of GVFS in https://github.com/lxqt/pcmanfm-qt/wiki

**Reply by stefonarch** [reply]
I agree on both; no idea why nm-tray is not packaged by arch, in debian and ubuntu it's available.

**Reply by stefonarch** [reply]
BTW I was thinking some weeks ago if we should have a direct link in PCmanfm-qt' help to the wiki. But the main problem is more than probably a majority of users don't understand english.

**Reply by tsujan** [reply]
Yes, English can be a problem. Apart from not knowing how it can be translated by Weblate, I'm not sure how a document that may change constantly can have an up-to-date translation. For example, right now I'm adding two sections to it.

Google-Translate my be the only solution.

EDIT: In an ideal world, another solution could be tell users to learn Esperanto and change the default language to it ;)

**Reply by stefonarch** [reply]
Esperanto had it's chance years ago... I remember my latin teacher being real furious about that idea ;)
As we discussed related to the html docs still in screengrab - weblate works [real well](https://translate.lxqt-project.org/projects/testing/screengrab-html/) with html. New or changed paragraphs will show up as untranslated. The problem is more where to display the html files, the links would be something like` ../it/docs.html /de/docs.html` , ideally pcmanfm-qt would parse locale and display a link (to lxqt-project.org/docs/pcmanfm.qt ?)
